hera_sim/visibilities/rimez_wrapper.py

```diff
@@ -0,0 +1,70 @@
+"""
+Wrapper around the healvis package for producing visibilities from
+healpix maps.
+"""
+
+import healpy
+import numpy as np
+from cached_property import cached_property
+from healvis.beam_model import AnalyticBeam
+
+from .simulators import VisibilitySimulator
+from astropy import constants as cnst
+import pyuvdata
+from pyuvsim.analyticbeam import AnalyticBeam
+from RIMEz import sky_models, beam_models, rime_funcs, utils, management
+
+
+class RIMEz(VisibilitySimulator):
+
+    def __init__(self,  **kwargs):
```

Comment:
Lets add a `NotImplementedError` in here for now

Reply:
Raised the error at the beginning of __init__.